app/tests/test_maths_service.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from app import maths_service
+
+
+class MathsServiceTests(unittest.TestCase):
+
+    def test_integer_coefficients_same_seed(self):
+        seed = 2000
+        coefficients_1 = maths_service.integer_coefficients(amount=3, seed=seed)
```

Comment:
This is cool but I think maybe hardcoded what values we expect to always see here?
So if we change how seeds work accidentally, the tests pick it up